Code_Exercises/Exercise_05_Device_Selection/README.md

```diff
@@ -23,23 +23,23 @@ using the `get_info` member function template.
 ### 2.) Try other device selectors
 
 Replace the default selector with one of the other standard device selectors
-that are provided by SYCL such as the `cpu_selector`, `gpu_selector` or
-`host_selector` and see which device those choose.
+that are provided by SYCL such as the `cpu_selector_v`, `gpu_selector_v` 
+and see which device those choose.
 
 ### 3.) Create your own device selector
 
-Create a device selector using the template below, implementing the function
+Create a device selector using the template below. implementing the function
```

Comment:
```suggestion
Create a device selector using the template below. Implement the function
```
Is this the intended change here? At the very least it's missing a capital but I think the flow is still strange.

Reply:
Done.